policy/modules/services/node_exporter.te

```diff
@@ -0,0 +1,62 @@
+policy_module(node_exporter)
+
+type node_exporter_t;
+type node_exporter_exec_t;
+init_daemon_domain(node_exporter_t, node_exporter_exec_t)
+
+type node_exporter_runtime_t;
+files_runtime_file(node_exporter_runtime_t)
+
+type node_exporter_var_lib_t;
+files_type(node_exporter_var_lib_t)
+
+type node_exporter_var_log_t;
```

Comment:
`node_explorer_log_t` instead. We're trying to avoid encoding the path in the type name.